### ISSUE TITLE
release: point wildcard tags to latest patch release

### ISSUE THREAD

**Issue by McShelby** [WyriHaximus/github-action-next-semvers#68]
Unlike WyriHaximus/github-action-create-milestone the wildcard tags like `v1` or `v1.2` do not point to their latest patch release (currently `v1.2.1`) making them unusable for actions.

**Comment by WyriHaximus** [comment]
It's a Docker based action so instead of using git tags to do that it happens on the image tag level. (Been working on a blog post about this.) Have a the tags on https://hub.docker.com/r/wyrihaximusgithubactions/next-semvers/tags they all share the same hash.

**Comment by McShelby** [comment]
Hm, I am not really a Docker man, so the magic is still unclear to me.

So let me rephrase into a question: Is there a way to set a wildcard release for this action? Will

      uses: WyriHaximus/github-action-next-semvers@v1

point to v1.2.1?

**Comment by WyriHaximus** [comment]
Yes. That is the whole going of using mutable Docker image tags. Working on a better way to communicate this in the release messages including links to the Docker images to clarify it. But in essence after tagging `v1.2.1`, a `docker tag v1.2.1 v1.2` and `docker tag v1.2.1 v1` which retags the `v1.2.1` as `v1.2` and `v1`.  It's pretty much the same way `WyriHaximus/github-action-create-milestone` works but instead of on the `git` level it works on the `Docker` level.

**Comment by McShelby** [comment]
Ah. Thanks for clarification. Feel free to close this issue or use it to mention this in the docs

**Comment by WyriHaximus** [comment]
Will keep it open until I put the clarification improvements here and in other actions that work this way are in place.
